shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1211,6 +1217,40 @@ - (void)invalidateTouchRateCorrectionVSyncClient {
   _touchRateCorrectionVSyncClient = nil;
 }
 
+#pragma mark - Stylus Events
+
+#pragma clang diagnostic push
+
+- (void)pencilInteractionDidTap:(UIPencilInteraction*)interaction API_AVAILABLE(ios(13.4)) {
+  flutter::PointerData pointer_data = [self generatePointerDataAtLastMouseLocation];
```

Comment:
do we need the mouse location for this pointer_data?

Reply:
looks like this is not resolved - we don't need the mouse location here. Can probably just do

```
  flutter::PointerData pointer_data;
  pointer_data.Clear();
```